package.json

```diff
@@ -23,18 +23,20 @@
     "async": "^1.3.0",
     "body-parser": "^1.13.1",
     "bower": "^1.4.1",
+    "cfenv": "~1.0.0",
     "chalk": "^1.1.0",
```

Comment:
@ilanbiala I think you're right. Looks like @trainerbill re-arranged to alphabetize the list.